Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -29,7 +29,7 @@ public func subscriptionConfirmation(
         style(maxWidth(.px(900)) <> margin(leftRight: .auto)),
       ],
       header(lane)
-        + (lane == .team ? teamMembers(currentUser, subscribeData) : [])
+        + teamMembers(lane: lane, currentUser: currentUser, subscribeData: subscribeData)
```

Comment:
I moved the `lane == .team` logic into this view function. I did this because in the case of individual subscriptions I need to render a hidden input (for `isOwnerTakingSeat`) and this view function seemed like a natural place to put it.

up for other ideas or if you think this function should be renamed!

Reply:
Nah I think this is fine for now.